docs/MO_DG/prepare_model/convert_model/Converting_Model.md

```diff
@@ -1,32 +1,52 @@
-# General Conversion Parameters {#openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model}
+# Setting Input Shapes {#openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model}
 
-To get the full list of general (framework-agnostic) conversion parameters available in Model Optimizer, run the following command:
+Paragraphs below provide details about specifing input shapes for model conversion.
+
+## When to Specify `--input_shape` Command Line Parameter <a name="when_to_specify_input_shapes"></a>
+There are situations when Model Optimizer is unable to deduce input shapes of the model. In this case, user has to specify input shapes explicitly
```

Comment:
This is partially out dated. We can freely work with undefined dimensions. And the problem should be sated as "we cannot always work with undefined input dimensions" and if you really run into this issue, you should specify it. Another reason is when we want to change the shape. Later should be references to dynamic shapes and reshape chapters in the runtime documentation. @jane-intel, please work with @rkazants to align on this stuff.

Reply:
Use static shapes when data size is determined and wouldn't be changed. So there will be no need to define it in run-time.
- User knows of data size and it is determined. So we can recommend to specify shapes for a model.
-

Reply:
"odel Optimizer is unable to deduce input shapes.." should also be mentioned

Reply:
Done